src/functions-and-arrays.js

```diff
@@ -1,19 +1,52 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(no1,no2) {
+  let number1 = no1
+  let number2 = no2
+
+  if(number1 > number2){
+    return number1
+  }
+  else if (number2 > number1){
+    return number2
+  }
```

Comment:
Ok, there is no need to declare variables `number1` and `number2` since already already passed `no1` and `no2`

You could have done just like this:

```
function maxOfTwoNumbers(num1, num2) {
  if (num1 > num2) return num1;
  else return num2;
}
```

This solution will return also `no2` when both arguments are equal joining the condition:
[should return either arguments - if both arguments are equal]